.github/workflows/backport.yml

```diff
@@ -14,14 +14,13 @@ jobs:
         with:
           repository: "grafana/grafana-github-actions"
           path: ./actions
-          # Pin the version to before https://github.com/grafana/grafana-github-actions/pull/113 because
-          # to avoid the strict rules for PR labels.
-          ref: d284afd314ca3625c23595e9f62b52d215ead7ce
```

Comment:
It would be nice to refer to a version, so that we have to update the workflow explicitly when new functionality is available? I suppose that now it'll just pull the latest from `main`?